ReflectometryServer/ioc_driver.py

```diff
@@ -315,16 +315,18 @@ def _get_component_sp(self):
         """
         Returns: position that the set point axis is set to
         """
-        if self._component.beam_path_set_point.axis[self._component_axis].is_in_beam:
+        if self._component.beam_path_set_point.is_in_beam:
```

Comment:
```suggestion
        if self._component.beam_path_set_point.is_in_beam or self._out_of_beam_lookup is None:
```

Reply:
If I put a axis without a park position out of the beam it now moves to zero. I think this is the right fix, but haven't tested it